plugins/foreman_ansible/1.x/index.md

```diff
@@ -0,0 +1,118 @@
+* FOREMAN_SSL_KEY: The private key when using SSL client certificates (default  "/etc/foreman/client_key.pem")
+* FOREMAN_SSL_VERIFY: wether to verify SSL certificates. Use *False*
+  to disable certificate checks. You can also set it to CA bundle (default is "True").
+
+To obtain a valid certificate for your host follow the procedure as described [here](http://theforeman.org/manuals/latest/index.html#4.3.10SSL)
+See the [python-requests documentation](http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification) on the details of certificate setup.
+
+## 2.2 Plugin
+
+If this is the first plugin you're installing, please see the [plugin
+repository section]({{site.baseurl}}plugins/#2.2Packageinstallation) to set up the
+repository first.
+
+To install Foreman Ansible, run the following:
+
+* Red Hat based (RHEL, CentOS...)
+
+      # yum install tfm-rubygem-foreman_ansible
+
+* Fedora
+
+      # yum install rubygem-foreman_ansible
+
+* Debian based (Debian, Ubuntu...)
+
+      # apt-get install ruby-foreman-ansible
+
+Restarting Foreman might be needed after installing the package.
+
+## 2.3 Inventory
+
+You can use the dynamic inventories feature in Ansible to make your inventory be Foreman, and have host properties and parameters available as Ansible variables in your manifests.
+
+The project is not packaged, but you can find the sources here at [theforeman/foreman_ansible_inventory](https://github.com/theforeman/foreman_ansible_inventory). Check Ansible's [dynamic inventory documentation](http://docs.ansible.com/ansible/intro_dynamic_inventory.html) to install it.
+
+# 3. Configuration
+
+In Foreman, you should add whatever Ansible hosts you want to submit facts from to the setting `trusted_puppetmaster_hosts`. Change it at Administer > Settings, Puppet tab.
+
+If the Foreman setting 'create_new_host_when_facts_are_uploaded' is true, and $HOSTNAME doesn't exist in Foreman, it will autocreate that host in Foreman. If it already exists, it will update the facts.
+
+Similarly, the Foreman setting 'ignore_puppet_facts_for_provisioning' is set to false, facts related to interfaces will update the interfaces of $HOSTNAME in Foreman.
+
+# 4. Usage
+
+Run ansible on your hosts and the callback will send everything to Foreman automatically. Here's a quick gif of what registering a new host in Foreman looks like.
+
+![registering a host](http://i.imgur.com/mlnVFJj.gif)
+
+If you want to assign Ansible roles to your hosts, please put them on `/etc/ansible/roles/`. After that the plugin will detect them and you will be able to assign them to your hosts when you edit them.
+
+![adding roles](http://i.imgur.com/UyeZIq8.png)
+
+To run the playbook with the roles selected, click on the 'play Ansible roles' button and Foreman will run a playbook on the background with these roles. Logs for this playbook can be found under the regular Foreman log, usually `/var/log/foreman/production.log`
+
+![role play](http://i.imgur.com/eU4RENK.png)
```

Comment:
It'd be better to keep these in the repository with the page, particularly as they're on HTTP URLs so will cause mixed content warnings.